packages/flutter_tools/lib/src/plugins.dart

```diff
@@ -891,7 +891,7 @@ Future<void> _writeIOSPluginRegistrant(FlutterProject project, List<Plugin> plug
   final List<Map<String, dynamic>> iosPlugins = _extractPlatformMaps(plugins, IOSPlugin.kConfigKey);
   final Map<String, dynamic> context = <String, dynamic>{
     'os': 'ios',
-    'deploymentTarget': '8.0',
+    'deploymentTarget': '9.0',
```

Comment:
This is not used for published plugins, it's generated for app and modules that need to import plugins.